packages/exec/src/interfaces.ts

```diff
@@ -30,6 +30,9 @@ export interface ExecOptions {
   /** optional. How long in ms to wait for STDIO streams to close after the exit event of the process before terminating. defaults to 10000 */
   delay?: number
 
+  /** optional. input to write to the process on STDIN. */
+  stdin?: Buffer
```

Comment:
I kept going back-and-forth about whether to accept a Buffer or Stream here. I think Buffer is a better API for most users, but it's not quite as flexible as giving users the ability to specify a stream.

Reply:
I agree that buffer is the better api for most users. Its simple and should work for most use cases here. If we end up implementing more complicated scenarios with streams in the future, we can always support passing in a stream or a buffer mutually exclusive. We some use cases for [that scenario](https://github.com/actions/toolkit/issues/346)

@bryanmacfarlane , FYI.

Reply:
Are we sure this is a buffer?  subprocess.stdin advertises it as a writable stream.  Another reason I wanted to see a pure js test instead of the script.  https://nodejs.org/api/child_process.html#child_process_subprocess_stdin

Reply:
The input we are allowing a user to pass in here is a buffer.

We write the contents of that buffer to the process's StdIn Stream.

Very similar to how node does this with the `input` option you outlined below, which also takes a string and various other inputs.
https://nodejs.org/api/child_process.html#child_process_child_process_execfilesync_file_args_options

Reply:
Sorry I just realized node also has `input` option. Should we just pass the value through and let node handle writing to the stream?

Reply:
...that said, seems like would be an implementation detail and wouldn't change behavior exposed by this PR. So i think this PR is OK the way it is (incremental)

Reply:
`input` doesn't appear to be available for any of the async process creation function. In particular we use spawn. https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options

Reply:
oh weird, didnt realize it was only on the sync functions. makes sense now why we dont pass thru :)